config/routes.rb

```diff
@@ -6,6 +6,9 @@
   
   get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
   get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
+  post '/users/:user_id/posts', to: 'posts#create', as: 'create_user_post'
+  post '/users/:user_id/posts/:id/comments', to: 'comments#create', as: 'create_user_post_comment'
+  get '/users/:user_id/posts/:id/likes', to: 'likes#create', as: 'create_user_post_like'
```

Comment:
- [ ] I think this route should have the method **POST** instead of **GET** with that the users will be able **to add Likes to Posts** as required for this project and also mentioned by previous reviewers :smile:  Please fix it in order to make your app works as expected :+1: 

##### HINT
```
   POST   /posts/:post_id/likes(.:format)     likes#create
```